pipeline/buildpacks/0.1/buildpacks.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: buildpacks
+  labels:
+    app.kubernetes.io/version: "0.1"
```

Comment:
Right now the CI doesn't tests the pipelines but can we still have some tests so that in future when we configure the CI to run on pipelines directory this pipeline can be tested?

Reply:
Could you elaborate on this a bit? I started looking at how difficult it would be to adopt the current e2e test scripts and it's not extremely straight forward. These tests that you are referring to, would they be much different from the samples that are being provided? Setting up a `pre-apply-task-hook.sh` doesn't translate to something usable to pipelines in its current form.

Reply:
Added a test from samples and created a `pre-apply-task-hook.sh` which hopefully will work once the testing harness is available.